vnext/Microsoft.ReactNative/IJSValueWriter.idl

```diff
@@ -1,24 +1,48 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#include "DocString.h"
+
 namespace Microsoft.ReactNative {
 
   // Writer for JSON-like streams or tree structures
   [webhosthidden]
+  DOC_STRING("``IJSValueWriter`` is used to read data from JavaScript in custom native modules.  It acts as a stream and supports all the types in [`JSValueType`](JSValueType.md).")
```

Comment:
Checking if double `` ` `` is intentional as other docs below only have a single one.

Reply:
markdown allows both single and double, I can change it to single :) FWIW this came from the website docs.